### PR TITLE
Implement clear command for the shell and fix a tiny bug in clearScreen

### PR DESCRIPTION
**Pull request by Malone-AI**
- Update handle_enter in Code/Kernel/Sources/keyboard.c to move cursor to the next line if it's not clear command
- Otherwise, clear screen and reset the cursor to the top-left corner
- Fixed clearScreen in Code/Kernel/Sources/vga.c to start with subscript 2 not 3

### CODE REVIEW

**Comment by IlanVinograd** [comment]
Thank you for the PR! I'll take a look at it as soon as I have some free time. I really appreciate your contribution!

**Comment by Malone-AI** [comment]
I am also very grateful to you for offering me such an opportunity to participate in your project.Have a good day!

**Comment by IlanVinograd** [comment]
Looks good, but on lines **179-180** inside the `handle_enter` function, there are commands for the shell that are implemented in the `shell.c/h` files to execute the appropriate shell commands. Could you add the `clean` command in `shell.c` and invoke it as a method below line 180 in handle_enter? Once that's done, I'll merge this PR.

Overall, everything looks great!

**Comment by Malone-AI** [comment]
Ok , I have understood your meanings. Thanks for your improvement suggestion . I will spend some time to finish it

**Comment by Malone-AI** [comment]
I think you are a genius, the code looks very beautiful and tidy as you suggested

File: Code/Kernel/Sources/keyboard.c [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #include "../Includes/keyboard.h"
+#include "../Includes/shell.h"
```
Comment (IlanVinograd) [inline Code/Kernel/Sources/keyboard.c]: No need, it was included in `keyboard.h` so you can remove it.

**Review by IlanVinograd — CHANGES_REQUESTED** [review]
Check `keyboard.c` file in review

File: Code/Kernel/Sources/keyboard.c (hunk 1 above)
Comment (Malone-AI) [reply Code/Kernel/Sources/keyboard.c]: Thank you for your patient guidance. I have removed it

**Review by IlanVinograd — APPROVED** [review]

**Pull request merged**